docs/modules/ROOT/pages/configure-cmos.adoc

```diff
@@ -0,0 +1,230 @@
+= Configuration Options
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+.Microlith configuration image
+image::microlith-runtime.png[]
+
+CMOS is designed as a microlith container with several services included in its stack.
+CMOS allows users to control these services in a fine grained manner.
+As shown in the above diagram, you can customize the installation by overriding the default service configuration values using environment variables or by providing a native configuration file for each service.
+
+== Disabling a Service
+
+All the services of CMOS stack are enabled by default.
+If you choose to disable a service, this is achieved by setting the corresponding environment variables of the docker container. The naming convention to be followed to disable a service is `DISABLE_X`, where `X` is the name of the service (e.g.,`DISABLE_ALERT_MANAGER`, `DISABLE_GRAFANA`, `DISABLE_JAEGER`, `DISABLE_LOKI`, `DISABLE_PROMETHEUS` and `DISABLE_WEBSERVER`).
+
+== Logging
+
+By default, logs can be either provided to standard output `(stdout)` or to service-based log files by setting the `ENABLE_LOG_TO_FILE` configuration value.
+If file output is used for the logs, these files will appear in the `/logs/` directory.
+Further, if desired, these logs can externally be shared via a volume or bind mount.
+
+To mount files into a container, use the `docker -v <source>:<destination>` command, ideally as read-only.
```

Comment:
If the end user wants the container to write the logs to the mounted path, it'll need to be mounted as read-write - so perhaps remove the last part of this sentence to avoid confusion?